components/brave_wallet_ui/components/desktop/chart-control-bar/style.ts

```diff
@@ -1,5 +1,9 @@
 import styled from 'styled-components'
 
+const isDarkTheme = (p: any) => {
+  return p.theme.name === 'Brave Dark'
+}
+
```

Comment:
I don't think this is a great way of detecting dark mode, since the theme name could be anything, especially if Wallet introduces extra wallet-specific properties.

If you really needed to detect dark mode in Javascript, you could use `window.matchMedia('(prefers-color-scheme: dark)') === true`, but I think the cleanest and most performant is to simply rely on the media query directly in the css:

```css
--selected-color: p.theme.palette.white
color: ${p => p.isSelected ? 'var(--selected-color)' : p.theme.color.text02};

@media (prefers-color-scheme: dark) {
  --selected-color: ${p => p.theme.color.background02};
}
```

Reply:
Removed isDarkTheme and is now using media query! :)